backend/server/src/db.rs

```diff
@@ -37,12 +37,31 @@ pub async fn create_pool(config: &config::Db) -> Result<Pool> {
         .context("failed to get DB connection")?;
     connection.execute("SELECT 1", &[]).await
         .context("failed to execute DB test query")?;
-    let n_roots = connection.execute("SELECT * from realms where id = 0", &[]).await
-        .context("failed to check")?;
-    if n_roots < 1 {
-        bail!("no root realm found");
-    } else if n_roots > 1 {
-        bail!("more than one root realm found");
+
+    // ensure database version exists
+    info!("Creating table version");
+    connection.execute(
+        "create table if not exists version (version int not null)", &[]).await
+        .context("failed to schema for table 'realms'")?;
```

Comment:
These kinds of things get formatted like this:

```suggestion
     connection
        .execute("create table if not exists version (version int not null)", &[])
        .await
        .context("failed to schema for table 'realms'")?;
```